src/router.ts

```diff
@@ -31,20 +31,57 @@ type UserOptions = {
 
 type AdminCommandParams = {
   commandParentMessageTs: string;
+  commandMessageTs: string; // if command is in a thread, this is the in-thread message
+  commandChannel: string;
   routingSlackUserName: string;
   previousSlackChannelName: string;
 };
 
+export function isStaleSession(userInfo: UserInfo): boolean {
+  if (!userInfo.sessionStartEpoch) {
+    // legacy session from before 2020-11-03 election
```

Comment:
It's specific to anything that predates this branch being merged, since starting now we'll populate the sessionStartEpoch when creating a new UserInfo.